.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
```

Comment:
I *think* the config can also live in pytest.ini (not sure)?  That'd help avoid ending with a gazillion config files in the root folder.  Also because it makes sense to keep that next to the pep8ignore list.

Reply:
flake8 is a superset of pep8 (renamed it pycodestyle at the BDFL's request) so if we run flake8, we no longer need to run pycodestyle.